packages/vue-app/template/utils.js

```diff
@@ -273,6 +273,13 @@ export function getQueryDiff(toQuery, fromQuery) {
   return diff
 }
 
+export function normalizeError(err) {
+  return {
+    message: err.message || (typeof err === 'string' ? err : JSON.stringify(err)),
```

Comment:
`JSON.stringify` may cause circular serialization problems. We are also in the browser context here and can't use big deps like `devalue`. Currently out of idea just mentioned that problem :D

Reply:
@pi0 but we use devalue in browser context :) But its not much usable for this usecase because it can produce not too readable js. I also though about https://bundlephobia.com/result?p=stringify-object@3.3.0 but as you said its still another dependency...

Reply:
@aldarund Sorry I was meaning that we don't bundle it with client dists. (Only used in template strings)

Reply:
@pi0 i added catch for serialization problems and it that case just display object name. That should be pretty rare case